apps/meteor/app/livechat/server/api/v1/visitor.ts

```diff
@@ -107,6 +107,11 @@ API.v1.addRoute(
 							errors.push(key);
 						}
 
+						const contacts = await LivechatContacts.findAllByVisitorId(visitor._id).toArray();
```

Comment:
having this logic here duplicated here is bad, we should unify the codes to prevent changes not being applied in both places in the future.. we can have a task to improve this later

it is duplicated from `setCustomFields` function:
https://github.com/RocketChat/Rocket.Chat/blob/1177d65ad979003c424b06e93af63ff4f64caa5f/apps/meteor/app/livechat/server/lib/custom-fields.ts#L36